config_examples/dblp_mapping.json

```diff
@@ -0,0 +1,47 @@
+{
+  "nodes": [
+    {
+      "nodeMappingId": "nm_author",
+      "nodeLabel": "Author",
+      "xpath": "/dblp/article/author",
+      "innerTextMappedTo": "name",
+      "mappedAttributes": {},
+      "mappedChildElements": {}
+    },
+    {
+      "nodeMappingId": "nm_publication",
+      "nodeLabel": "Publication",
+      "xpath": "/dblp/article",
+      "innerTextMappedTo": null,
+      "mappedAttributes": {
+        "publtype": "type"
+      },
+      "mappedChildElements": {
+        "title": "title",
+        "year": "year"
+      }
+    }
+  ],
+  "edges": [
+    {
+      "edgeLabel": "CollaboratedWith",
+      "directed": false,
+      "firstNodeMappingId": "nm_author",
+      "secondNodeMappingId": "nm_author",
```

Comment:
Skoro mają być tylko krawędzie rodzic-dziecko i dziecko-dziecko, to nie łatwiej byłoby podać ścieżkę do rodzica, tag dziecka i rodzaj krawędzi? Można się pozbyć Id wtedy.